studio/stores/authConfig/schema/AuthProviders/AuthProvidersFormValidation.tsx

```diff
@@ -300,80 +304,139 @@ const EXTERNAL_PROVIDER_APPLE = {
   title: 'Apple',
   properties: {
     EXTERNAL_APPLE_ENABLED: {
-      title: 'Enable Apple provider',
-      description: 'This will enable Apple login for your application',
+      title: 'Enable Sign in with Apple',
+      description:
+        'Enables Sign in with Apple on the web using OAuth or natively within iOS, macOS, watchOS or tvOS apps.',
       type: 'boolean',
     },
     EXTERNAL_APPLE_CLIENT_ID: {
-      /**
-       * to do: change docs
-       */
-      title: 'Services ID',
-      description: `
-Client identifier when authenticating or validating users.
+      title: 'Service ID (for OAuth)',
+      description: `Client identifier used in the OAuth flow on the web.
 [Learn more](https://developer.apple.com/documentation/sign_in_with_apple/configuring_your_environment_for_sign_in_with_apple)`,
       type: 'string',
     },
-    EXTERNAL_IOS_BUNDLE_ID: {
-      /**
-       * to do: change docs
-       */
-      title: 'IOS Bundle ID',
-      description: `The iOS app's unique identifier. [Learn more](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids)`,
-      type: 'string',
-    },
     EXTERNAL_APPLE_SECRET: {
-      /**
-       * to do: change docs
-       */
-      title: 'Secret key',
-      description: `
-The secret key is a JWT token that must be generated.
+      title: 'Secret Key (for OAuth)',
+      description: `Secret key used in the OAuth flow.
 [Learn more](https://supabase.com/docs/guides/auth/social-login/auth-apple#generate-a-client_secret)`,
       type: 'string',
       isSecret: true,
     },
+    EXTERNAL_APPLE_ADDITIONAL_CLIENT_IDS: {
+      title: 'Authorized Client IDs (iOS, macOS, watchOS, tvOS bundle IDs or service IDs)',
+      description: `Comma separated list of allowed Apple app bundle IDs for native sign in, or service IDs for Sign in with Apple JS. [Learn more]()`,
```

Comment:
```suggestion
      description: `Comma separated list of allowed Apple app bundle IDs for native sign in, or service IDs for Sign in with Apple JS. [Learn more](https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_js)`,
```
Should there be a link in the parens? Provided a placeholder, feel free to change as needed